### PR TITLE
Update documentation for serviceAccount Key and WorkloadIdentity

### PR DESCRIPTION
**Pull request by emmanuelstroem**
This PR is intended to clarify the setup of Google Service Account and its relation to the `Service Account Key` and `Workload Identity` options.
It also adds some links within sections and to the helm chart. 

This points to the policies step as a place to add the permission.

#### Change:
- Required step for Google Service Account with permission to run Velero.
- Optional steps for setup using either serviceAccount Key or workload Identity

#### Fix:
- Bash script variable references.
- Links to sections within README.
- Link to [Helm chart](https://github.com/vmware-tanzu/helm-charts)

### CODE REVIEW

**Review by sseago — COMMENTED** [review]
I don't have extensive GCP experience with velero, but one part about the wording here confuses me, although this confusion may predate your PR. From the doc, there are two different methods recommended for setting permissions ("Option 1" and "Option 2"). If you're following "Option 2", then you shouldn't have to perform steps from "Option 1", but the "Option 2" instructions refer back to something created near the end of the "Option 1" workflow. That being said, even without this change, it looks like this discrepancy already existed, just without "Option 1" being called out explicitly. Is there a way to reword this to avoid that confusion?

**Comment by emmanuelstroem** [comment]
@sseago 
You are definitely correct about the confusion. 
Both options need a GGP role and ServiceAccount in place. 

Option 1 - then creates a serviceAccount key
Option 2 - utilises workload identity. 

I will reword the PR to clarify that 🙏

**Comment by emmanuelstroem** [comment]
@sseago and @zubron 
Sorry for the most delayed update. 🙏
Would really appreciate if you can take a look. 😄

**Comment by emmanuelstroem** [comment]
@sseago 
would you be so kind as to give this PR another check? 🙏

File: README.md [hunk 1]
```diff
@@ -156,7 +187,7 @@ For more information on configuring workload identity on GKE, look at the [offic
 
 Install Velero, including all prerequisites, into the cluster and start the deployment. This will create a namespace called `velero`, and place a deployment named `velero` in it.
 
-**If using a Service Account**:
+**If using a Googl Service Account Key**:
```
Comment (sseago) [inline README.md:190]: nit: s/Googl/Google/

**Review by sseago — DISMISSED** [review]
lgtm with one typo to correct (noted in review comment)

File: README.md (hunk 1 above)
Comment (emmanuelstroem) [reply README.md]: Thanks for the catch @sseago . . . Pushed a fix for the type 🙏

**Comment by emmanuelstroem** [comment]
@zubron  Happy New Year \o/ 

Sorry for being pushy on this, would really appreciate a review from you before workloadIdentity gets deprecated 🙈 😄

**Review by jenting — APPROVED** [review]
